packages/marketing/src/nps/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormEvent } from 'react';
+
+function NpsScore( score: number ) {
+	return (
+		<label>
+			<input type="radio" name="nps-score" value={ score } />
+			{ score }
+		</label>
+	);
+}
+
+export function Nps() {
```

Comment:
Once we get a fully working version, we can keep this as the plain/view only component (that we can also build Stories for), and ship the wrapper as Nps that would also include the handlers for showing/not showing, mutating, etc.